docs/examples/example_globcurrent.py

```diff
@@ -51,15 +55,15 @@ def set_globcurrent_fieldset(
     "use_xarray", [True, pytest.param(False, marks=pytest.mark.xfail)]
 )
 def test_globcurrent_fieldset(use_xarray):
-    fieldset = set_globcurrent_fieldset(use_xarray=use_xarray)
```

Comment:
why is this not tested anymore now?

Reply:
The globcurrent dataset has some vars with `attrs["Unit"]` instead of `attrs["units"]` hindering cf decoding. This is very easy to patch if we use xarray (done in this PR for this test case), but that forces `set_globcurrent_fieldset()` to have to use xarray.